common/src/main/java/com/wynntils/functions/generic/MathFunctions.java

```diff
@@ -0,0 +1,161 @@
+    }
+
+    public static class DivideFunction extends GenericFunction<Double> {
+        @Override
+        public Double getValue(FunctionArguments arguments) {
+            return arguments.getArgument("first").getDoubleValue()
+                    / arguments.getArgument("second").getDoubleValue();
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.Builder(List.of(
+                    new FunctionArguments.Argument<>("first", Number.class, null),
+                    new FunctionArguments.Argument<>("second", Number.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("div");
+        }
+    }
+
+    public static class ModuloFunction extends GenericFunction<Double> {
+        @Override
+        public Double getValue(FunctionArguments arguments) {
+            return arguments.getArgument("first").getDoubleValue()
+                    % arguments.getArgument("second").getDoubleValue();
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.Builder(List.of(
+                    new FunctionArguments.Argument<>("first", Number.class, null),
+                    new FunctionArguments.Argument<>("second", Number.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("mod");
+        }
+    }
+
+    public static class PowerFunction extends GenericFunction<Double> {
+        @Override
+        public Double getValue(FunctionArguments arguments) {
+            return Math.pow(
+                    arguments.getArgument("first").getDoubleValue(),
+                    arguments.getArgument("second").getDoubleValue());
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.Builder(List.of(
+                    new FunctionArguments.Argument<>("first", Number.class, null),
+                    new FunctionArguments.Argument<>("second", Number.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("pow");
+        }
+    }
+
+    public static class SquareRootFunction extends GenericFunction<Double> {
+        @Override
+        public Double getValue(FunctionArguments arguments) {
+            return Math.sqrt(arguments.getArgument("value").getDoubleValue());
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.Builder(
+                    List.of(new FunctionArguments.Argument<>("value", Number.class, null)));
+        }
+
+        @Override
+        public List<String> getAliases() {
+            return List.of("sqrt");
+        }
+    }
+
+    public static class RoundFunction extends GenericFunction<Double> {
```

Comment:
I think you also need an `int` function that rounds to the nearest integer, and returns the result as an `Integer`. So if you e.g. force the modulo function to take int as the second argument, the user can use the `int` function to make sure they send valid values.

Reply:
(At first I thought that round should return int but then I realize this has a slightly different meaning, which is totally fine, doing it an int is a different task)

Reply:
I added it.